src/index.html

```diff
@@ -10,7 +10,43 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Search bar airbnb</h1>
+  <body class="body">
+    <form
+      class="form"
+      action="#"
+      method="get"
+      data-qa="big"
+    >
+      <img
+        class="icon-big"
+        src="images/Search.svg"
+        alt="search"
+      >
+      <input
```

Comment:
Don't forget to add empty lines between multiline siblings
```suggestion
      >
      
      <input
```